seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -1954,7 +1954,8 @@
         "function myFunction() {\n  var myVar;\n  console.log(myVar);\n}\nmyFunction();"
       ],
       "tests": [
-        "assert(code.match(/console\\.log/gi).length === 1, 'message: Remove the second console log');"
+        "assert(typeof myVar === 'undefined', 'message: No global <code>myVar</code> variable');",
+        "assert(/var myVar/.test(code), 'message: Add a local <code>myVar</code> variable');"
```

Comment:
You'll probably need to check for any amount of whitespace between var and myVar as `var       myVar;` is syntacticly valid.